hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -2650,11 +2650,26 @@
       tests: []
    }
 
+    ////////////////////////
+    // Mask ROM Functests //
+    ////////////////////////
+    {
+      name: chip_sw_mask_rom_functests
+      desc: '''Run functests developed for mask ROM.
+
+            Mask ROM functests are developed by the SW team to test mask ROM
+            components (e.g., drivers, libraries, etc.) work as intended
+            on-chip. However, unlike the mask ROM, these tests are linked with
+            the OTTF, and run out of flash, after being booted by the test ROM.
+            They should work on all platforms, including in DV.
+            '''
+      milestone: V2
+      tests: ["chip_sw_mask_rom_keymgr_functest"]
```

Comment:
If there are 100s of functests, will they all be added to this testpoint? This seems too generic to me. I would think there should be a keymgr functest in the ROM testplan, which is what this would get mapped to instead.

Reply:
There are only ~10 functests, but good point, I moved this to the mask ROM testplan Hjson (and renamed the `mask_rom_e2e_testplan.hjson` to `mask_rom_testplan.hjson` since it contains more than just mask ROM E2E tests now.